tests/main/snap-quota-groups/task.yaml

```diff
@@ -4,6 +4,15 @@ details: |
   Functional test for snap quota group commands ensuring that they are 
   effective in practice.
 
+# Memory accounting is not functional without workarounds on these old systemd
```

Comment:
(nitpick) to be on the safe side we could change the version check so that we don't even attempt to run on centos7/azl2 but this is probably ok for now too as long as we get back to this and either fix or disable it.

Reply:
option 1 from this PR description does exactly this if we decide that's what we want

Reply:
sorry that is to say, I have the commit doing just that in a followup PR that we can choose from after this PR - I would rather leave this PR as-is and handle that in a followup if that's what we want

Reply:
Makes sense, let's land this so that we can get to the next option then.